### DISCUSSION TITLE
How to have multiple documentation menu

### DISCUSSION THREAD

**Discussion by mashuq** [thuliteio/doks#262] [Q&A]
I would like to have 2 documentation menu in main menu, just like https://getdoks.org/ site, which has "docs" and 'tutorial'

but when I download the theme, it has only 1 main menu which hosts documents "docs". I could not make another documentation menu. Please help.

**Comment by MattDodsonEnglish** [comment]
If I'm not mistaken, the responses in this thread should help you fix the problem. https://github.com/h-enk/doks/discussions/173 
H-enk even made a pull request, so you could review the commits there and modify them how you want.

You'll need to change `config/_default/menus.toml` some stuff in `layouts`, and a few other things.
